### ISSUE TITLE
clock sends stop signal when timer reaches zero

### ISSUE THREAD

**Issue by kmeesters** [FirstLegoLeague/clock#15]
The clock changes (visually) to a stop state when it reaches zero. This could also be posted on the message bus so other applications know the time has run out. For example for lights or displaysystems.

**Comment by rikkertkoppes** [comment]
We can easily add that. However, it creates the school example of a race condition when there are multiple clock instance.

You'd need an additional effort on the receiving side to handle that properly.

**Comment by kmeesters** [comment]
Yes, perhaps we could implement something along the lines of what we discussed for scoring? There would be clock 'master' who holds the actual countdown time and send a stop signal. This would also prevent other client on the message bus to accidentally start / reset / arm the clock? This could be also only a rule when a clock has been running (ie. start signal send from a certain node, can only be stopped by that node, or overriden (confirmation)

**Comment by rikkertkoppes** [comment]
I'd rather see a [rp approach](http://rxmarbles.com/#distinctUntilChanged). Combine the streams, pass the result on
